react/components/ProductImages/index.js

```diff
@@ -38,23 +39,31 @@ const ProductImages = ({
     hiddenImages && hiddenImages.map(text => new RegExp(text, 'i'))
   const handles = useCssHandles(CSS_HANDLES)
 
-  const images = allImages
-    .filter(
-      image =>
-        !image.imageLabel ||
-        !excludeImageRegexes.some(regex => regex.test(image.imageLabel))
-    )
-    .map(image => ({
-      type: 'image',
-      url: image.imageUrls ? image.imageUrls[0] : image.imageUrl,
-      alt: image.imageText,
-      thumbUrl: image.thumbnailUrl || image.imageUrl,
-    }))
-  const videos = allVideos.map(video => ({
-    type: 'video',
-    src: video.videoUrl,
-    thumbWidth: 300,
-  }))
+  const images =
+    contentType !== 'videos'
+      ? allImages
+          .filter(
+            image =>
+              !image.imageLabel ||
+              !excludeImageRegexes.some(regex => regex.test(image.imageLabel))
+          )
+          .map(image => ({
+            type: 'image',
+            url: image.imageUrls ? image.imageUrls[0] : image.imageUrl,
+            alt: image.imageText,
+            thumbUrl: image.thumbnailUrl || image.imageUrl,
+          }))
+      : []
+
+  const videos =
+    contentType !== 'images'
```

Comment:
The same idea applies here. 

```suggestion
    shouldIncludeVideos
```

Reply:
Another advantage to abstracting this into a variable is that we could make the conditional more complex in the future (maybe when another prop is added), and this part of the code, and the understanding of it, would remain the same 😃 .

Reply:
Yeah, it looks good to me too. Thanks @victorhmp .